frontend/src/components/Forms/SignUpForm/SignUpForm.jsx

```diff
@@ -40,10 +44,22 @@ export const SignUpForm = () => {
 	};
 
 	const onSubmit = (data, form) => {
+		console.log('submitted signupform, data is', data)
+		// what about an event prevent default? Do we need to set the form data or just send it to the server?
```

Comment:
I didn't seem to need a prevent default when I was testing, but that's something we can add and play around with. I think we need to set the form data with this kind of form.